crates/db/src/entities/bridge_tx_state.rs

```diff
@@ -101,18 +101,18 @@ impl BridgeTxState {
     }
 
     /// Get the private nonce for the transaction being tracked in this state.
-    pub fn secnonce(&self) -> &Musig2SecNonce {
+    pub const fn secnonce(&self) -> &Musig2SecNonce {
         &self.secnonce
     }
```

Comment:
This is a really big one, it would definitely be wrong to have a SecNonce is a const context.  But probably also all of the other accessors here.